public/js/extraFunctions.js

```diff
@@ -654,6 +654,143 @@ function tableData_closeAllSelect(elmnt) {
 
 }
 
+let initialize_resize_hamburger = function() {
+
+  //total width of all items in tab minus hamburger
```

Comment:
Add spacing and capitalization in comments. In this case it would be the following, but you would need to make the change on all of your comments.
```suggestion
  // Total width of all items in tab minus hamburger
```